jekyll/_cci2/browser-testing.md

```diff
@@ -122,12 +122,12 @@ If you are running a test that runs an HTTP server on CircleCI, it is sometimes
 failing test. Setting this up is easy with an SSH-enabled
 run.
 
-1. Run an SSH build using the Rerun Job with SSH button on the {% comment %} TODO: Jobs {% endcomment %}Builds page of the CircleCI app. The command to log into
+1. Run an SSH build using the Rerun Job with SSH button on the **Jobs page** of the CircleCI app. The command to log into
```

Comment:
Same should-this-be-a-link question. I'm going to stop leaving this comment. If it _should_ be a link, just be sure to grep for other instances of "Jobs page"